pcdsdevices/attenuator.py

```diff
@@ -105,25 +106,38 @@ class AttBase(FltMvInterface, PVPositioner):
     vary. You should not instantiate this class directly, but instead use the
     :func:`Attenuator` factory function.
     """
-
+    # fundamental frequency components
+    # ================================
     # Positioner Signals
     setpoint = Cpt(EpicsSignal, ':COM:R_DES', auto_monitor=True,
                    kind='normal')
     readback = Cpt(EpicsSignalRO, ':COM:R_CUR', auto_monitor=True,
                    kind='hinted')
-    actuate = Cpt(EpicsSignal, ':COM:GO', kind='omitted')
```

Comment:
I'm going to change these back to `omitted` - just noticed that I left them here, but this was just for testing